sonar-java-plugin/src/main/java/org/sonar/plugins/java/JavaPlugin.java

```diff
@@ -44,7 +43,7 @@ public void define(Context context) {
       builder.add(JavaSonarLintClasspath.class);
     } else {
       builder.addAll(SurefireExtensions.getExtensions());
-      builder.addAll(JaCoCoExtensions.getExtensions());
+      //builder.addAll(JaCoCoExtensions.getExtensions());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)